common/vetDisplay.c

```diff
@@ -48,8 +49,8 @@ void sendAmountToDisplayString(txInt256_t *sendAmount, uint8_t *ticker, uint8_t
 }
 
 void maxFeeToDisplayString(txInt256_t *gaspricecoef, txInt256_t *gas, feeComputationContext_t *feeComputationContext, uint8_t *displayString) {
-    convertUint256BE(MAX_GAS_COEF, sizeof(MAX_GAS_COEF), &feeComputationContext->maxGasCoef);
-    convertUint256BE(BASE_GAS_PRICE, sizeof(BASE_GAS_PRICE), &feeComputationContext->baseGasPrice);
+    convertUint256BE((uint8_t *)MAX_GAS_COEF, sizeof(MAX_GAS_COEF), &feeComputationContext->maxGasCoef);
```

Comment:
Maybe we could change the parameter type of `convertUint256BE` to take a const instead of dropping the const though a cast here?

Reply:
As this function doesn't change the value pointed by it's first parameter it should be made `const`.
Casting from non `const` to `const` is not an issue, and compiler won't even request it, so you will not have to do it. But the other way can be dangerous.

Reply:
ok agree.